src/v2/Apps/Home/Components/HomeFeaturedEventsRail.tsx

```diff
@@ -138,13 +138,21 @@ export const HomeFeaturedEventsRailFragmentContainer = createFragmentContainer(
             subtitle
             href
             image {
-              small: cropped(width: 95, height: 63) {
+              small: cropped(
+                width: 95
+                height: 63
+                version: ["main", "wide", "large_rectangle"]
+              ) {
                 src
                 srcSet
                 width
                 height
               }
-              large: cropped(width: 445, height: 297) {
+              large: cropped(
+                width: 445
+                height: 297
+                version: ["main", "wide", "large_rectangle"]
```

Comment:
Interesting - i've seen this around but unsure what it does?

Reply:
Chooses which version to process — images aren't guaranteed to have all the versions. In this case it was defaulting to `large_rectangle` which is actually pretty small. So we're saying here to prefer main, if that's not available, then go to wide, then to large_rectangle.